test/e2e/service.go

```diff
@@ -966,6 +968,89 @@ var _ = framework.KubeDescribe("Services", func() {
 			framework.Failf("expected un-ready endpoint for Service %v within %v, stdout: %v", t.name, kubeProxyLagTimeout, stdout)
 		}
 	})
+
+	It("should be able to create services of type LoadBalancer and externalTraffic=localOnly [Slow][Feature:externalTrafficLocalOnly]", func() {
```

Comment:
@girishkalele looks like the pattern used elsewhere is Feature:CapitalFeatureName, so this should be `[Feature:ExternalTrafficLocalOnly]`. Can you fix and update in [job spec](https://github.com/kubernetes/test-infra/blob/master/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml#L165)?